src/main/java/cinema/config/AppConfig.java

```diff
@@ -15,7 +15,8 @@
 @ComponentScan(basePackages = {
         "cinema.service",
         "cinema.dao",
-        "cinema.security"
+        "cinema.security",
+        "cinema.dto"
```

Comment:
why are these packages under spring control?

Reply:
```suggestion
        "cinema.security",
```
Требование в прошлом PR-e